README_dev.md

```diff
@@ -2,107 +2,97 @@
 
 1. Download and install [Node JS](https://nodejs.org/en/download/)
 2. run `npm i`
-3. run `npm run build:watch` (this is also added as a VSCode task that is configured to [run on folder open](https://code.visualstudio.com/updates/v1_30#_run-on-folder-open))
+3. run `npm run -ws build`
+
+Optionally, run `npm run build:watch`. This is also added as a VSCode task that is configured to [run on folder open](https://code.visualstudio.com/updates/v1_30#_run-on-folder-open).
```

Comment:
I'm not a fan of this change - to really be able to work with this monorepo you should enable build:watch .. I don't see it as optional.